source/github-helpers/timeline-item.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'dom-chef';
 
-export default function TimelineItem(): JSX.Element {
+export function TimelineItemOld(): JSX.Element {
 	// Classes copied from #issuecomment-new + mt-3 added
 	return <div className="ml-0 pl-0 ml-md-6 pl-md-3 mt-3" />;
 }
+
+// #8141
```

Comment:
I've been preferring full URLs unless it's just minor information. Here you left no other comment so this makes it easier to get more context.

```suggestion
// https://github.com/refined-github/refined-github/pull/8141
```